### PR TITLE
Copy interplay into main repository

### PR DESCRIPTION
**Pull request by ihostage**
As part of https://github.com/playframework/interplay/issues/150

### CODE REVIEW

File: project/Playdoc.scala [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * Copyright (C) from 2022 The Play Framework Contributors <https://github.com/playframework>, 2011-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+import sbt._
+import sbt.io.IO
+import sbt.Keys._
+
+object Playdoc extends AutoPlugin {
+
+  final val Docs = config("docs")
```
Comment (mkurz) [inline project/Playdoc.scala:11]: I see you moved that out of `autoImport` :wink: I think I understand why you did that, shouldn't matter anyway.

Comment (ihostage) [reply project/Playdoc.scala:11]: Yep, one warning scared me 😄

**Review by mkurz — APPROVED** [review]
OK, let's merge that now so we don't have to mess with interplay in 2.9 and 3.0 anymore.

I saw you removed 
https://github.com/playframework/interplay/blob/ad1bddb900e2883f7ad78e59c70fc86603903a52/src/main/scala/interplay/PlayBuildBase.scala#L39-L42

but we never needed that in this repo.

Maybe in future we can even simplify this setup, but let's just use the copy approach for now.

I tested this PR by locally publishing the artifcts with and without interplay and basically the exacct same output gets generated. I even unpacked all the jar files wiht a script to diff everything. Just to be sure we don't introduce a problem anymore.

**Comment by mkurz** [comment]
BTW, I switched back to the `._` wildcards for imports because we still use the Scala 2 syntax everyhwere.

I will also take a quick look at https://github.com/playframework/play-doc/pull/206 because Play depends on that so we can upgrade play-docs as well so we are completely interplay free here.

I will after that create a `2.9.x` branch to merge the Pekko PR. We are basically done now.

**Pull request merged**